concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2334,6 +2335,7 @@ jobs:
                       'DATADOG_APP_KEY' => '${DATADOG_APP_KEY:-}',
                       'DEPLOY_ENV' => '${DEPLOY_ENV}',
                       'ELB_ADDRESS' => 'https://healthcheck.${APPS_DNS_ZONE_NAME}/',
+                      'TLS_DOMAINS' => 'healthcheck.${APPS_DNS_ZONE_NAME},api.${SYSTEM_DNS_ZONE_NAME}',
```

Comment:
We can't just cover a single subdomain as the certificate should be valid for the parent domain as well. This is because the CDN broker-created CloudFront Distributions use the parent domain as the origin by default and is HTTPS only. We should probably add the parent domain (just `$APPS_DNS_ZONE_NAME`) as well.